en-US/Design.xml

```diff
@@ -542,7 +542,7 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
 </example>
 
           <para>
-			For omitting vertical columns of output, use a consistent notation such as an ellipsis (...).
+			For omitting vertical columns of output, or omitting part of horizontal lines of output, Red&nbsp;Hat uses the ellipsis notation (...).
```

Comment:
```suggestion
			For omitting vertical columns of output, omitting partial lines of output, or to shorten long strings in output, Red&nbsp;Hat uses the ellipsis notation (...).
```

Reply:
The Confluence documentation we recently discusses is about the specific use case of shortening long strings. My suggested edit clarifies this use case -- but this is a soft suggestion only.